aws-serverless-java-container-struts2/pom.xml

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>aws-serverless-java-container-struts2</artifactId>
+    <name>AWS Serverless Java container support - Struts2 implementation</name>
+    <description>Allows Java applications written for the Struts2 framework to run in AWS Lambda</description>
+    <url>https://aws.amazon.com/lambda</url>
+    <version>1.2-SNAPSHOT</version>
+
+    <parent>
+        <groupId>com.amazonaws.serverless</groupId>
+        <artifactId>aws-serverless-java-container</artifactId>
+        <version>1.2-SNAPSHOT</version>
+    </parent>
+
+    <properties>
+        <struts2.version>2.5.16</struts2.version>
+        <jackson.version>2.9.5</jackson.version>
+    </properties>
+
+    <dependencies>
+        <!-- Core interfaces for the aws-serverless-java-container project -->
+        <dependency>
+            <groupId>com.amazonaws.serverless</groupId>
+            <artifactId>aws-serverless-java-container-core</artifactId>
+            <version>1.2-SNAPSHOT</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.struts</groupId>
+            <artifactId>struts2-core</artifactId>
+            <version>${struts2.version}</version>
+            <optional>true</optional>
```

Comment:
Is this optional because you expect developers to pick their own version of Struts2? If we want to enable this, we should make sure that this can support most versions of struts 2.

Reply:
You are right struts2-core should be required when this lib is used. I changed this.